src/_data/catalog/index.json

```diff
@@ -29,6 +29,39 @@
       }
     ],
     "id": "material"
+  },  
+  {
+    "name": "Material 3 Components",
+    "description": "Visual, behavioral, and motion-rich widgets implementing the <a href=\"https://material.io/design/guidelines-overview\">Material 3 Design guidelines</a>.<br /><br />Material 3 will become the default in late 2023 - see <a href=\"https://m3.material.io/develop/flutter\">m3.material.io/develop/flutter</a> for more details.",
```

Comment:
suggestion to link to https://flutter.github.io/samples/web/material_3_demo/ and point to useMaterial3 flag

Reply:
Sure - due to the templating structure, I've added this as a `content` field, so it shows up in the page but not that card.

Reply:
Done, thanks!